From 3a79e5714a38873aed9ad1847f77c4e2710c458b Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Sat, 9 Mar 2013 19:24:29 +0000
Subject: [PATCH 32/35] EPEL 5: ruby: Use old rake/rdoctask and rake/gempackagetask.

The 'begin ... rescue' code is supposed to ensure these old packages
are required, but for reasons not understood this doesn't work on
ancient Ruby 1.8.5 in RHEL 5.
---
 ruby/Rakefile.in |   15 +++------------
 1 files changed, 3 insertions(+), 12 deletions(-)

diff --git a/ruby/Rakefile.in b/ruby/Rakefile.in
index a74e191..fbf8ccc 100644
--- a/ruby/Rakefile.in
+++ b/ruby/Rakefile.in
@@ -20,21 +20,12 @@ require 'rake/clean'
 require 'rake/testtask'
 
 # Used to be rake/rdoctask.  Now it's rdoc/task.
-begin
-  require 'rdoc/task'
-rescue
-  require 'rake/rdoctask'
-end
+require 'rake/rdoctask'
 
 # Used to be rake/gempackagetask.  Now it's rubygems/package_task.  Also
 # we need to use the appropriate class name below.
-begin
-  require 'rubygems/package_task'
-  gempackagetask='Gem::PackageTask'
-rescue
-  require 'rake/gempackagetask'
-  gempackagetask='Rake::GemPackageTask'
-end
+require 'rake/gempackagetask'
+gempackagetask='Rake::GemPackageTask'
 
 PKG_NAME='@PACKAGE_NAME@'
 PKG_VERSION='@PACKAGE_VERSION@'
-- 
1.7.4.1

